hedera-node/src/main/java/com/hedera/services/ledger/TransactionalLedger.java

```diff
@@ -295,7 +302,23 @@ public void commit() {
     /** {@inheritDoc} */
     @Override
     public boolean exists(final K id) {
-        return existsOrIsPendingCreation(id) && !isZombie(id);
+        final var unaliasedId = resolveIfAlias(id);
```

Comment:
I don't understand why this is necessary---doesn't `AliasResolver` replace all known aliases, and then `AutoCreationLogic` replace all requested auto-creations in the `BalanceChange` objects?

Reply:
yeah agreed. I think I added it for some reason before completely implementing `AutoCreationLogic`

Reply:
Ah great news. 😁